app/Offer/OfferControllerProvider.php

```diff
@@ -60,6 +61,8 @@ public function connect(Application $app): ControllerCollection
 
         $controllers->put('/{offerType}/{offerId}/organizer/{organizerId}/', UpdateOrganizerRequestHandler::class);
 
+        $controllers->patch('/{offerType}/{offerId}', PatchOfferRequestHandler::class);
```

Comment:
```suggestion
        $controllers->patch('/{offerType}/{offerId}/', PatchOfferRequestHandler::class);
```

We always add a trailing slash, and I think it's actually needed to make the request work with both a trailing slash and without a trailing slash.